### PR TITLE
WT-11269 Account for would-be aborted in-memory updates in RTS dry run

### PR DESCRIPTION
**Pull request by pmacko86**

### CODE REVIEW

File: src/include/btmem.h [hunk 1]
```diff
@@ -1424,13 +1424,14 @@ struct __wt_update {
...
 /* When introducing a new flag, consider adding it to WT_UPDATE_SELECT_FOR_DS. */
 /* AUTOMATIC FLAG VALUE GENERATION START 0 */
-#define WT_UPDATE_DS 0x01u                       /* Update has been written to the data store. */
-#define WT_UPDATE_HS 0x02u                       /* Update has been written to history store. */
-#define WT_UPDATE_PREPARE_RESTORED_FROM_DS 0x04u /* Prepared update restored from data store. */
-#define WT_UPDATE_RESTORED_FAST_TRUNCATE 0x08u   /* Fast truncate instantiation */
-#define WT_UPDATE_RESTORED_FROM_DS 0x10u         /* Update restored from data store. */
-#define WT_UPDATE_RESTORED_FROM_HS 0x20u         /* Update restored from history store. */
-#define WT_UPDATE_TO_DELETE_FROM_HS 0x40u        /* Update needs to be deleted from history store */
+#define WT_UPDATE_DRYRUN_WOULD_ABORT 0x01u       /* Would be aborted, if not for dry run. */
```
Comment (quchenhao) [inline src/include/btmem.h:1427]: May be rename to WT_UPDATE_RTS_DRYRUN_ABORT?

Comment (pmacko86) [reply src/include/btmem.h]: Good idea—I like that name better! Changed in [6ad9b13](https://github.com/wiredtiger/wiredtiger/pull/10518/commits/6ad9b13d0812cdaff838890b82cc370f5dd1d720).

**Review by quchenhao — APPROVED** [review]
lgtm

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**